openvdb/openvdb/tools/Diagnostics.h

```diff
@@ -441,7 +442,7 @@ struct CheckNormGrad
         : acc(grid.getConstAccessor())
         , invdx2(ValueType(1.0/math::Pow2(grid.voxelSize()[0])))
         , minVal2(_min*_min)
-        , maxVal2(_max*_max)
+        , maxVal2(_max < std::numeric_limits<ValueType>::max() ? _max*_max : _max)
```

Comment:
I think we should focus this PR on only divbyzero and invalid FPEs. This change doesn't appear to prevent overflows anyway? a max value of "numeric limits max - 1" will still overflow here.

Reply:
I will remove it, per your suggestion, although I do disagree: It does not catch all overflows, but does catch the most prevalent one where `FLT_MAX` is used as un-initialized value. Not only is it common, it is actually occurring in the existing test suite!

Reply:
Also, taking it out, means that #2090 bug is not fixed.